### PR TITLE
12.0 wip product_stock_state

### PR DESCRIPTION
**Pull request by kevinkhao**

### CODE REVIEW

File: product_stock_state/views/res_config_settings_view.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <record id="res_config_settings_view_form_threshold" model="ir.ui.view">
+            <field name="name">res.config.settings.threshold</field>
+            <field name="model">res.config.settings</field>
+            <field name="inherit_id" ref="stock.res_config_settings_view_form"/>
+            <field name="priority" eval="31"/>
+            <field name="arch" type="xml">
+                <xpath expr="//div[@data-key='stock']//div[hasclass('o_settings_container')][3]" position="inside">
+                    <!--AAAAAAAAAAinsideAAAAAAAAAAAAAAAAAAAAAA-->
+                    <div class="col-12 col-lg-6 o_setting_box">
+                            qqqqqqqqqqqqqqqqqqqqqqqqqqqqq
```
Comment (sebastienbeau) [inline product_stock_state/views/res_config_settings_view.xml]: need cleanup

File: product_stock_state/views/res_config_settings_view.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                                </div>
+                            </div>
+
+                        </div>
+                    </div>
+                </xpath>
+            </field>
+        </record>
+
+        <!--<div id="production_lot_info" position="after">-->
```
Comment (sebastienbeau) [inline product_stock_state/views/res_config_settings_view.xml]: need cleanup

**Review by sebastienbeau — CHANGES_REQUESTED** [review]
Travis is red please fix it

**Comment by rousseldenis** [comment]
@sebastienbeau Even if we have easier access to this repo, I suggest to move those generic modules like 'product_...' in ... the OCA product repository as it can fullfill other users requirements and could be maintained by those others too.

**Comment by sebastienbeau** [comment]
@rousseldenis good idea !
@kevinkhao can you do a PR here : https://github.com/OCA/product-attribute ?
Thanks

**Comment by kevinkhao** [comment]
sure @sebastienbeau https://github.com/OCA/product-attribute/pull/575

**Comment by PierrickBrun** [comment]
I think we should close this PR

**Comment by hparfr** [comment]
Closed because new home for this module will be OCA/product-attribute#575

**Pull request closed without merging**